.github/workflows/update-helm-repo.yaml

```diff
@@ -218,7 +218,7 @@ jobs:
           git tag "${TAGNAME}"
 
       - name: Make github release
-        uses: softprops/action-gh-release@b21b43df682dab285bf5146c1955e7f3560805f8 # tags/v1
+        uses: softprops/action-gh-release@de2c0eb89ae2a093876385947365aca7b0e5f844 # tags/v1
```

Comment:
this was recently changed, [here](https://github.com/grafana/helm-charts/commit/0041b4f6729f11023cd3fd305108cef7cff28710#diff-31cb5c1bb5a66990c71c62c0a16b5eefa85482e630470ff008b3f741d7189846L206-R221) in https://github.com/grafana/helm-charts/commit/0041b4f6729f11023cd3fd305108cef7cff28710:

```diff
-- uses: softprops/action-gh-release@v1
++ uses: softprops/action-gh-release@b21b43df682dab285bf5146c1955e7f3560805f8 # tags/v1
```

however, commit https://github.com/softprops/action-gh-release/commit/b21b43df682dab285bf5146c1955e7f3560805f8 points to a commit from ~6 years ago, whereas https://github.com/softprops/action-gh-release/commit/de2c0eb89ae2a093876385947365aca7b0e5f844 is the commit associated with [the v1 tag](https://github.com/softprops/action-gh-release/releases/tag/v1)